docs/src/pages/components/pagination/PaginationRanges.js

```diff
@@ -15,6 +15,9 @@ export default function PaginationRanges() {
 
   return (
     <div className={classes.root}>
+      <Pagination count={11} defaultPage={6} siblingCount={0} boundaryCount={0} />
```

Comment:
Is this a breaking change or why do we need to update the docs?

Reply:
Should just need a test...

Reply:
> Thanks for working on this.
> 
> Could you include tests in `packages/material-ui-lab/src/Pagination/Pagination.test.js` that were failing before this change?

Just added this test.

Reply:
> Is this a breaking change or why do we need to update the docs?

I've added this because the argos test would fail if it were not working. The unit test is enough for this case.